Kong/docs.konghq.com
# [DOCU-3028] Clean up runtime instance install topics

### Description

Combed through the runtime instance installation topics and pulled out a couple of things:
* how to verify that a runtime instance is running
* the list of resources that get created in AWS
* kubernetes troubleshooting info

Deleted all runtime installation topics, including the initially planned advanced Docker instructions. Since the script was replaced in the UI, those instructions are no longer necessary. 

In general, all of the instructions we had in the docs are now in the UI, and are no longer necessary to maintain in the documentation.

https://konghq.atlassian.net/browse/DOCU-3028

### Testing instructions

Netlify links:
https://deploy-preview-5349--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/
https://deploy-preview-5349--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/verify-instance/
https://deploy-preview-5349--kongdocs.netlify.app/konnect/runtime-manager/troubleshoot/#kubernetes-runtime-instance-installation-does-not-work

### Checklist
